recipes/texture-atlas/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: texture-atlas
+  version: {{ environ['GIT_DESCRIBE_TAG'] ~ "." ~ environ['GIT_DESCRIBE_NUMBER'] ~ "." ~ environ['GIT_DESCRIBE_HASH'] }}
+
+source:
+#  git_rev: {{ sha }}
+  git_branch: add-findproj4
+  git_url: git@gitlab.kitware.com:danlipsa/core3d-keu.git
+  patches:
+    - gdal-headers.patch
+
+requirements:
+  build:
```

Comment:
`cmake` and `ninja` should in this list.

Reply:
What is the convention here? Should cmake and ninja be considered system provided that same way libc, X and OpenGL is? I would rather not specify that for every package. I did try installing the conda cmake and I got an error when launching ccmake. But I did not spend too much time on it.
Note the vtk conda forge package does not specify those

https://github.com/conda-forge/vtk-feedstock/blob/master/recipe/build.sh

Reply:
If they are not installed on the system then the package won't build. For example, I'm testing by building on a clean virtual machine.

The VTK conda-forge package does specify `cmake` and `ninja` as build requirements:
https://github.com/conda-forge/vtk-feedstock/blob/3df978152ca4f3a648b7e962f5ded7e2a6346c70/recipe/meta.yaml#L26-L27

Reply:
I see. I missed that. Thanks. I'll add the same.